cmd/evm/runner.go

```diff
@@ -62,7 +62,7 @@ var runCommand = cli.Command{
 
 // readGenesis will read the given JSON format genesis file and return
 // the initialized Genesis structure
-func readGenesis(genesisPath string) *types.Genesis {
+func ReadGenesis(genesisPath string) *types.Genesis {
```

Comment:
Yeah, it's not necessary now but it might be in the future. There is a similar function in go-ethereum and there it is exported. If you still feel it's better not to change it I will update the PR.